packages/x-data-grid/src/internals/demo/TailwindDemoContainer.tsx

```diff
@@ -0,0 +1,21 @@
+import * as React from 'react';
+
+interface TailwindDemoContainerProps {
+  children: React.ReactNode;
+}
+
+/**
+ * WARNING: This is an internal component used in documentation to inject the Tailwind script.
+ * Please do not use it in your application.
+ */
+export function TailwindDemoContainer(props: TailwindDemoContainerProps) {
+  const { children } = props;
+
+  React.useEffect(() => {
+    const script = document.createElement('script');
+    script.src = 'https://cdn.tailwindcss.com';
+    document.body.appendChild(script);
+  }, []);
```

Comment:
This is a temporary solution - the tailwind CDN not intended for production builds. Will be removed before merge.